app/tasks/__init__.py

```diff
@@ -46,6 +58,62 @@ def send_async_email(email,
     )
 
 
+
+@celery.on_after_configure.connect
+def setup_periodic_tasks(**kwargs):
+    # Calls updateSchedulerevery at midnight
+    celery.add_periodic_task(crontab(minute=0, hour=0), updateScheduler.s(), name='check credits expiry')
+
+@celery.task()
+def updateScheduler():
+    
+    app = current_app
+
+    db = SQLAlchemy()
+
+    db.create_all()
+
+    print("Executing task : Checking credit expiry...")
+    credit_assignment_schema = CreditAssignmentSchema(many = True)
+
+    todays_datetime = datetime(datetime.today().year, datetime.today().month, datetime.today().day)
+    #         SELECT c.user_id, c.amount,  u.expiry_date
```

Comment:
I believe these commented out code can now be removed

Reply:
Sounds good. Thanks